axelrod/load_data_.py

```diff
@@ -0,0 +1,45 @@
+import pkg_resources
+
+
+def load_file(filename, directory):
```

Comment:
Wouldn't these be better as a set of pandas dataframes? There would be far less code and it would be quicker too.

I know it's another dependency, but we're already dependent on numpy, so the precedent has been set.